test/specs/custom-bundling-roots/custom-roots.spec.js

```diff
@@ -285,6 +285,113 @@ describe("Custom bundling roots", () => {
       });
     });
 
+    it("given collision, should append mid to the key", async () => {
+      const defaults = createStoplightDefaults(__dirname, "http://localhost:8080/api/nodes.raw/", "gh/stoplightio/test");
+
+      nock("http://localhost:8080")
```

Comment:
Urgh. I'll get rid of `nock` in some other PR, as browser tests cannot be run due to it.